spec.html

```diff
@@ -33717,14 +33693,13 @@ <h1>%TypedArray%.prototype.some ( _callbackfn_ [ , _thisArg_ ] )</h1>
         <h1>%TypedArray%.prototype.sort ( _comparefn_ )</h1>
         <p>%TypedArray%`.prototype.sort` is a distinct function that, except as described below, implements the same requirements as those of `Array.prototype.sort` as defined in <emu-xref href="#sec-array.prototype.sort"></emu-xref>. The implementation of the %TypedArray%`.prototype.sort` specification may be optimized with the knowledge that the *this* value is an object that has a fixed length and whose <emu-xref href="#integer-index">integer-indexed</emu-xref> properties are not sparse. The only internal methods of the *this* value that the algorithm may call are [[Get]] and [[Set]].</p>
```

Comment:
> The only internal methods of the *this* value that the algorithm may call are [[Get]] and [[Set]].

Isn't this violated by these changes?

Reply:
The hole deletion loop doesn't get hit for TypedArrays, as far as I can tell, so we shouldn't have to worry about that part.

Reply:
I think we should remove the

> The only internal methods of the *this* value that the algorithm may call are [[Get]] and [[Set]].

bit, since that was only relevant when implementations had considerably more freedom for how to do the sort. Now that the reading and writing parts are fully specified it's not relevant.

Reply:
Whatever way it goes, it would still be nice to use an abstract op instead of magic step replacement.

Reply:
I've made @bakkot's suggested change. W.r.t. using an abstract operation, I agree, but since it's orthogonal to this PR I'd prefer to tackle it separately.